src/utils/test.ts

```diff
@@ -7,3 +7,15 @@ describe("getAutoGeneratedCommentForPath Function", () => {
     expect(result).to.equal("# Source files");
   });
 });
+describe("getAutoGeneratedCommentForPath Function", () => {
+  it("Should return empty string for non-common folder", () => {
+    const result = getAutoGeneratedCommentForPath("sssssssss");
+    expect(result).to.equal("");
+  });
+});
+describe("getAutoGeneratedCommentForPath Function", () => {
+  it("Should return empty string for empty string", () => {
+    const result = getAutoGeneratedCommentForPath("");
+    expect(result).to.equal("");
+  });
```

Comment:
@sahandfarahani @Manpreet-Bhatti One Describe Per Function. Multiple IT per Function.